lib/pacemaker/Makefile.am

```diff
@@ -10,21 +10,21 @@ include $(top_srcdir)/Makefile.common
 AM_CPPFLAGS	+= -I$(top_builddir) -I$(top_srcdir)
 
 ## libraries
-lib_LTLIBRARIES	= libpacemaker-internal.la
+lib_LTLIBRARIES	= libpacemaker.la
 
 ## SOURCES
 
-libpacemaker_internal_la_LDFLAGS	= -version-info 1:0:0
+libpacemaker_la_LDFLAGS	= -version-info 1:0:0
 
-libpacemaker_internal_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
-libpacemaker_internal_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
+libpacemaker_la_CFLAGS	= $(CFLAGS_HARDENED_LIB)
+libpacemaker_la_LDFLAGS	+= $(LDFLAGS_HARDENED_LIB)
 
-libpacemaker_internal_la_LIBADD	= $(top_builddir)/lib/pengine/libpe_status.la \
+libpacemaker_la_LIBADD	= $(top_builddir)/lib/pengine/libpe_status.la \
 			$(top_builddir)/lib/cib/libcib.la \
 			$(top_builddir)/lib/transition/libtransitioner.la \
 			$(top_builddir)/lib/lrmd/liblrmd.la
 # -L$(top_builddir)/lib/pils -lpils -export-dynamic -module -avoid-version
-libpacemaker_internal_la_SOURCES	= pcmki_sched_allocate.c \
+libpacemaker_la_SOURCES	= pcmki_sched_allocate.c \
```

Comment:
I'd abandon the pcmki prefix for source code files and just use pcmk

Reply:
If we're using the same prefix for everything, maybe it makes sense to not use a prefix at all and instead just make sure things are in directories to keep them organized.

Reply:
A pcmk_ prefix could still distinguish library source files from daemon/tool source files. But as long as the daemons have unique prefixes, I suppose it doesn't matter if the libraries have one, as long as we avoid the daemon prefixes (like sched in this case).

The real goal is unique file names across the project, for when using PCMK_trace_files and reading log messages with trace logging turned on (which shows file names). When you have logs like:

pacemaker.log-20180429.gz:Apr 27 21:44:22 [7525] rhel7-1 pacemaker-schedulerd: (     utils.c:559   )   debug: custom_action:	rhel7-1	Fencing_start_0 (cancelled : quorum)

we might as well not log the file name. The 1.1 code base has 8 files named utils.c. :)

Alternatively we could keep the idea of unique prefixes per library, use pcmk_ for libpacemaker, and try to come up with something imaginative when we reorganize the others.